ui/client/README.md

```diff
@@ -1,41 +1,115 @@
 # Prerequisites
 
+## NodeJS and NPM 
+
 Make sure you have relatively new nodejs/npm version (see `.nvmrc` for details)
 
+## Package dependencies
+
+Fetch locked package dependencies using:
+```
+npm ci 
+```
+
 # Run
 
+You have a few options to provide backend for frontend needs. After each "run" command, frontend will be available at http://localhost:3000. 
+Below there are described possible options.
+
+## Using backend started with docker
+
+This option requires docker and connection to docker hub registry. It will use `touk/nussknacker` image in `staging-latest` version.
 ```
-npm ci && npm start
-open http://localhost:3000
+npm run start:backend-staging
 ```
 
+## Using locally started backend
+
+This option uses backend started at http://localhost:8080 . For more information how to start it, take a look at [Backend instruction](../README.md) 
```

Comment:
remove space before dot .

Reply:
I don't like adding interpunction symbols to machine purpose links. Maybe it will work on generated html but some other tool can have a problem with this.